lesson_07/objects/objects_app/src/main/java/com/codedifferently/lesson7/JavyenWare/Book.java

```diff
@@ -0,0 +1,97 @@
+package com.codedifferently.lesson7;
```

Comment:
Also, the package name should be `com.codedifferently.lesson7.javyenware` (all lowercase). You should update the folder name as well to lowercase (here and for your test file).